cli/cmd/project/refresh.go

```diff
@@ -113,6 +116,13 @@ func RefreshCmd(ch *cmdutil.Helper) *cobra.Command {
 				return fmt.Errorf("the --full flag can only be used with --all or --model")
 			}
 
+			// Return an error for ineffective use of --partition
+			if len(modelPartitions) > 0 {
+				if !cmd.Flags().Changed("model") {
+					return fmt.Errorf("--partition requires --model to be explicitly set")
+				}
+			}
```

Comment:
How can this case ever happen? Isn't it already covered by the check on line 86?